rctab/routers/accounting/send_emails.py

```diff
@@ -49,17 +49,12 @@ async def get_sub_email_recipients(
 ) -> List[str]:
     """Returns the users we should email about this subscription.
 
-    Parameters
-    ----------
-    database : Database
-        a database to keep a record of a subscription and its users
-    subscription_id : UUID
-        a subscription ID
+    Args:
+        database : a database to keep a record of a subscription and its users.
+        subscription_id : a subscription ID.
 
-    Returns
-    -------
-    List[str]
-        contains email addresses of the users that should receive the email
+    Returns:
+        A string that contains email addresses of the users that should receive the email.
```

Comment:
This function returns a list of strings:

```suggestion
        The email addresses of the users that should receive the email.
```